apps/site/assets/ts/schedule/components/ScheduleLoader.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { ReactElement } from "react";
 import { connect } from "react-redux";
-import { useQueryParams, StringParam } from "use-query-params";
```

Comment:
I initially tried updating `use-query-params`, but its updated version required wrapping the relevant React components in a `<Provider>` type component, which I was wary to do. Also our use case is very simple, so it felt like we could get away without the overhead.